rpm/SPECS/couchdb.spec

```diff
@@ -33,29 +33,49 @@ ExclusiveArch: x86_64
 Exclusiveos:   linux
 Packager:      CouchDB Developers <dev@couchdb.apache.org>
 
+%if 0%{?suse_version}
+BuildRequires: erlang
+BuildRequires: erlang-rebar
+BuildRequires: erlang-reltool
+BuildRequires: erlang-epmd
+BuildRequires: gcc-c++
+BuildRequires: pkg-config
+Requires(pre): shadow
+BuildRequires: python3
```

Comment:
This is not correct. Many of the python scripts in 2.2.0 are not yet python 3 compatible.

This will be remedied with https://github.com/apache/couchdb/pull/1639 and should arrive in 2.3.0. I can either wait to merge this until 2.3.0 is released, or you can change this (and the following lines) to python2 and wait to file another PR when 2.3.0 comes out.

Reply:
Also, see https://github.com/apache/couchdb-ci/pull/8

Reply:
The original file has `python >= 2.6`. openSUSE has two python packages: `python2` and `python3`. My decision was `python3`. I will change it to `python2`

Reply:
No problem. I changed again "python2" to "python3" in `couchdb.spec`. I look forward to 2.3.0 full of expectation. Thanks for your work.